WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -164,6 +194,10 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
             title: NSLocalizedString("Disconnect from WordPress.com", comment: "Label for disconnecting from WordPress.com account"),
             action: confirmLogout())
 
+        let addAccount = ButtonRow(
+            title:  NSLocalizedString("Add WP Account", comment: "Add account for WordPress.com"),
```

Comment:
I don't think "WP" is an abbreviation we use in user facing labels. This should say "Add WordPress.com account"